src/Dhall/Import.hs

```diff
@@ -134,6 +134,7 @@ import Data.Text.Lazy.Builder (Builder)
 import Data.Traversable (traverse)
 #endif
 import Data.Typeable (Typeable)
+import qualified Filesystem.Path.CurrentOS
```

Comment:
For alignment reasons I try to keep all the qualified imports together at the bottom

Reply:
Fixed in patch.